odoo_install.sh

```diff
@@ -345,6 +348,21 @@ EOF
 else
   echo "Nginx isn't installed due to choice of the user!"
 fi
+
+#--------------------------------------------------
+# Enable ssl with certbot
+#--------------------------------------------------
+
+if [ $INSTALL_NGINX = "True" ] && [ $ENABLE_SSL = "True" ] && [ $ADMIN_EMAIL != 'odoo@example.com' ];then
+  sudo add-apt-repository ppa:certbot/certbot -y && sudo apt-get update -y
+  sudo apt-get install python-certbot-nginx -y
+  sudo certbot --nginx -d $WEBSITE_NAME --noninteractive --agree-tos --email $ADMIN_EMAIL --redirect
+  sudo service nginx reload
+  echo "SSL/HTTPS is enabled!"
+else
+  echo "SSL/HTTPS isn't enabled due to choice of the user!"
```

Comment:
Since you also have a check for `[ $ADMIN_EMAIL != 'odoo@example.com' ]` this else might give invalid text.
Maybe split it up in another  condition or make the message say `SSL/HTTPS isn't enabled due to choice of the user or because of a misconfiguration!`